docs/parent/pom.xml

```diff
@@ -46,10 +45,24 @@
     </properties>
 
     <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <artifactId>maven-deploy-plugin</artifactId>
+                    <executions>
+                        <execution>
+                            <id>default-deploy</id>
+                            <phase>disabled</phase>
+                        </execution>
+                    </executions>
+                </plugin>
+            </plugins>
+        </pluginManagement>
         <plugins>
             <plugin>
                 <groupId>org.glassfish.doc</groupId>
                 <artifactId>glassfish-doc-maven-plugin</artifactId>
+                <version>1.3</version>
```

Comment:
All these plugins are inherited to all children, they don't have any customizations, so it doesn't bring any value. That is why I left versions here. The only plugin where I had doubts is the git scm plugin, which could be even in GF parent pom as in the future it could be used by the maven release plugin.

Reply:
Just FYI, I tried to do that locally and I have to confirm what I wrote - it just inflated the pom without any value.
It would make sense it the configuration and executions would change between modules but here it is quite simple situation, rare in GlassFish :-)

Reply:
Sure, I know. It’s OK as it is now.